seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/serialize/source/DefaultSeaTunnelRowDeserializer.java

```diff
@@ -233,7 +233,9 @@ Object convertValue(SeaTunnelDataType<?> fieldType, String fieldValue)
     private LocalDateTime parseDate(String fieldValue) {
         // handle strings of timestamp type
         try {
-            long ts = Long.parseLong(fieldValue);
+            // ES timestamps are allowed to be in epoch seconds, we should uniformly convert them to
+            // epoch milliseconds
+            long ts = Long.parseLong(fieldValue) * (fieldValue.length() == 10 ? 1000 : 1);
```

Comment:
why can get date scale by https://github.com/apache/seatunnel/blob/875fcd89c59faa4c7b97f3e2c1bf3be707a205cd/seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/catalog/ElasticSearchTypeConverter.java#L134. Can we use this information to convert date not use length?